src/main/java/com/cloudbees/jenkins/support/filter/ContentFilters.java

```diff
@@ -68,4 +71,12 @@ public void setEnabled(boolean enabled) {
         return Messages.ContentFilters_DisplayName();
     }
 
+    /**
+     * 
+     * @return the global configuration category for CasC where this config lands
+     */
+    @Override
+    public @Nonnull GlobalConfigurationCategory getCategory() {
```

Comment:
I think you need to override another method for older versions? Maybe I'm wrong but worth to have a look.

Reply:
The method you were thinking on should be https://github.com/jenkinsci/support-core-plugin/pull/211/files#diff-1055300cec105daa6e56921490b6795aR254 but it's for the new categorized UI, different from JCasC categorization. Precisely I took this PR to find out where we should place the JCasC configuration, the same as where it will land on the new categorized UI.